doc/tutorial/Server.lhs

```diff
@@ -37,7 +36,6 @@ import Data.Maybe
 import Data.String.Conversions
 import Data.Time.Calendar
 import GHC.Generics
-import Lucid
```

Comment:
We actually do need the Lucid import I think.  As we're using all sorts of lucid combinators later on.

The https://github.com/haskell-servant/servant/tree/master/doc/tutorial  directory is just a cabal project for all the literate haskell files.    So that way you can check if the code works.